tests/test_admin_client.py

```diff
@@ -307,6 +307,30 @@ async def test_list_subscriptions(
     ]
 
 
+async def test_list_subscriptions_empty_response(
+    app: aiohttp.web.Application,
+    admin_client: Callable[[], Awaitable[AdminClient]],
+    recorder: Recorder,
+) -> None:
+    """Test listing subscriptions."""
+
+    handler = NewHandler(
+        recorder,
+        [{"subscriptions": []}],
```

Comment:
Is this actually exercising the bug? I think the bug is when the response is `{}`

Reply:
Ahh yep you're right! 


The original error is a `KeyError` with the `subscriptions` key missing from the response:
```python
return response["subscriptions"]
```

Meaning that the `response` json could either:
1. Have other properties except `subscriptions` in it
or
2. Be an empty json

Thanks for checking my logic on that 🙌🏿
I'll go ahead and commit that change